### DISCUSSION TITLE
debug mode

### DISCUSSION THREAD

**Discussion by tophlcvf** [lloesche/valheim-server-docker#463] [Q&A]
i make myself as admin  ( use docker on synology).   i can us F5 and basic admin commands but not the debug mode.  is there any admin lvl or something like that ?
  thx

**Comment by drpcken** [comment]
I believe, [to accomplish what you are looking for](https://github.com/lloesche/valheim-server-docker#admin-commands) you'll need to add your [steamid ](https://www.steamidfinder.com/) to the `/config/adminlist.txt` file of your server.  Then [enable the console](https://github.com/lloesche/valheim-server-docker#enable-admin-console) for your game in the steam launch parameters.

**Reply by tophlcvf** [reply]
i did that, and i can use the consol, basic command. but i can t the debug mode .  maybe a anticheat to on and off somewhere

**Reply by drpcken** [reply]
I think you'll need a separate mod to do that.  Check "Quality of Life" or "SKToolbox" mods.

**Reply by tophlcvf** [reply]
i will try,  but with the new update, bepinex don't work anymore

thx for helping
